Dockerfile

```diff
@@ -1,13 +1,9 @@
 FROM python:3.9.16-slim-buster AS builder-image
+ENV PYTHONUNBUFFERED=1
 
-# install requirements
-COPY requirements.txt .
-RUN pip install --no-cache-dir -r requirements.txt
-
-COPY ./src /src
-WORKDIR /src
+COPY . /code
+WORKDIR /code
 
-# make sure all messages always reach console
-ENV PYTHONUNBUFFERED=1
+CMD pip install .
 
-ENTRYPOINT ["python3", "__main__.py"]
+ENTRYPOINT ["zorak"]
```

Comment:
This entrypoint seems to be not working as expected. 

![image](https://github.com/practical-python-org/ZorakBot/assets/57622136/7ec61103-ff77-4103-917a-b628ec46734d)